test/role_test.js

```diff
@@ -68,11 +74,15 @@ suite('api (roles)', function() {
       description: 'test prefix role',
       scopes: ['dummy-scope-2'],
     });
+
+    await helper.events.waitFor('e1');
+
     assume(role.description).equals('test prefix role');
     assume(new Date(role.created).getTime()).is.atmost(Date.now());
     assume(role.scopes).deep.equals(['dummy-scope-2']);
     assume(role.expandedScopes).contains('dummy-scope-2');
-    assume(role.expandedScopes).does.not.contain('assume:' + roleId);
+    // expandedScopes should always include itself
+    assume(role.expandedScopes).contains('assume:' + roleId);
```

Comment:
I suspect that this used to pass due to a race.  *Everything* else consistently tests that role.expandedScopes contains `assume:${role.roleId}`.